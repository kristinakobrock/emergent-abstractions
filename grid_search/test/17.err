  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:02, 22.39it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 25.91it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 25.29it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 23.47it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 26.42it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 26.01it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 24.74it/s] 50%|##############################################                                              | 25/50 [00:00<00:00, 26.00it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 23.65it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 25.54it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 25.54it/s] 76%|#####################################################################9                      | 38/50 [00:01<00:00, 26.61it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 28.65it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 27.68it/s] 96%|########################################################################################3   | 48/50 [00:01<00:00, 25.81it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.40it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 25.86it/s] 54%|##################################################                                           | 7/13 [00:00<00:00, 30.77it/s] 85%|#############################################################################8              | 11/13 [00:00<00:00, 27.04it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.89it/s]
