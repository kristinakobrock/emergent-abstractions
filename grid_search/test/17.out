# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6670063138008118, "acc": 0.6187264323234558, "length": 3.579360246658325, "mode": "train", "epoch": 1}
{"loss": 0.6614278554916382, "acc": 0.6197265982627869, "length": 4.0, "mode": "test", "epoch": 1}
{"loss": 0.6601768136024475, "acc": 0.6303194761276245, "length": 3.623993158340454, "mode": "train", "epoch": 2}
{"loss": 0.656059980392456, "acc": 0.6253905892372131, "length": 4.0, "mode": "test", "epoch": 2}
{"loss": 0.6582460999488831, "acc": 0.629153847694397, "length": 3.6533701419830322, "mode": "train", "epoch": 3}
{"loss": 0.6575738787651062, "acc": 0.6226562857627869, "length": 4.0, "mode": "test", "epoch": 3}
{"loss": 0.6574835181236267, "acc": 0.6305537819862366, "length": 3.66131329536438, "mode": "train", "epoch": 4}
{"loss": 0.6573576331138611, "acc": 0.6228516101837158, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.6568485498428345, "acc": 0.6305040717124939, "length": 3.712261199951172, "mode": "train", "epoch": 5}
{"loss": 0.6541190147399902, "acc": 0.6253905892372131, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.657055675983429, "acc": 0.6305954456329346, "length": 3.7481062412261963, "mode": "train", "epoch": 6}
{"loss": 0.6503651142120361, "acc": 0.6240234971046448, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.6555999517440796, "acc": 0.6307613253593445, "length": 3.801453113555908, "mode": "train", "epoch": 7}
{"loss": 0.6485292315483093, "acc": 0.6296875476837158, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.6531802415847778, "acc": 0.631153404712677, "length": 3.8482120037078857, "mode": "train", "epoch": 8}
{"loss": 0.6471966505050659, "acc": 0.6283202767372131, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.6488481163978577, "acc": 0.6322695016860962, "length": 3.872199296951294, "mode": "train", "epoch": 9}
{"loss": 0.6425089240074158, "acc": 0.6310546398162842, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.6409691572189331, "acc": 0.6394426822662354, "length": 3.90191912651062, "mode": "train", "epoch": 10}
{"loss": 0.6340265274047852, "acc": 0.6419921517372131, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.6293107867240906, "acc": 0.6498019099235535, "length": 3.934138298034668, "mode": "train", "epoch": 11}
{"loss": 0.6203440427780151, "acc": 0.6634765267372131, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.6132473349571228, "acc": 0.6675124764442444, "length": 3.9609062671661377, "mode": "train", "epoch": 12}
{"loss": 0.6180167198181152, "acc": 0.66796875, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.6001280546188354, "acc": 0.680790901184082, "length": 3.974675178527832, "mode": "train", "epoch": 13}
{"loss": 0.6234917640686035, "acc": 0.6726561188697815, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.5956574082374573, "acc": 0.686313807964325, "length": 3.9821577072143555, "mode": "train", "epoch": 14}
{"loss": 0.6286579370498657, "acc": 0.6757811903953552, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.5873483419418335, "acc": 0.6931084990501404, "length": 3.984659433364868, "mode": "train", "epoch": 15}
{"loss": 0.6227366328239441, "acc": 0.6843750476837158, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.5803238153457642, "acc": 0.6968056559562683, "length": 3.985004425048828, "mode": "train", "epoch": 16}
{"loss": 0.6154755353927612, "acc": 0.6857423186302185, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.5756068229675293, "acc": 0.7034101486206055, "length": 3.9846291542053223, "mode": "train", "epoch": 17}
{"loss": 0.60881108045578, "acc": 0.6910156607627869, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.5681138038635254, "acc": 0.7139002084732056, "length": 3.986856460571289, "mode": "train", "epoch": 18}
{"loss": 0.5953709483146667, "acc": 0.7005859017372131, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.5590799450874329, "acc": 0.7188792824745178, "length": 3.9855453968048096, "mode": "train", "epoch": 19}
{"loss": 0.594855546951294, "acc": 0.7103515267372131, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.5498382449150085, "acc": 0.7254357933998108, "length": 3.9835119247436523, "mode": "train", "epoch": 20}
{"loss": 0.5934540033340454, "acc": 0.7093750238418579, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.5396456122398376, "acc": 0.7332088947296143, "length": 3.98299241065979, "mode": "train", "epoch": 21}
{"loss": 0.5938308238983154, "acc": 0.7050781846046448, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.5325842499732971, "acc": 0.7355029582977295, "length": 3.9822068214416504, "mode": "train", "epoch": 22}
{"loss": 0.5940184593200684, "acc": 0.7046875357627869, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.5289747714996338, "acc": 0.7413478493690491, "length": 3.9853498935699463, "mode": "train", "epoch": 23}
{"loss": 0.5950441956520081, "acc": 0.6958984732627869, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.5213767290115356, "acc": 0.7452853322029114, "length": 3.9849843978881836, "mode": "train", "epoch": 24}
{"loss": 0.5824830532073975, "acc": 0.6996093392372131, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.5166115164756775, "acc": 0.7479254007339478, "length": 3.9878058433532715, "mode": "train", "epoch": 25}
{"loss": 0.5811339616775513, "acc": 0.7128906846046448, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.5109817385673523, "acc": 0.7531872391700745, "length": 3.9887399673461914, "mode": "train", "epoch": 26}
{"loss": 0.5759998559951782, "acc": 0.7123046517372131, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.5051442384719849, "acc": 0.7617802023887634, "length": 3.9883785247802734, "mode": "train", "epoch": 27}
{"loss": 0.5578492879867554, "acc": 0.7314453721046448, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.5003953576087952, "acc": 0.7625153660774231, "length": 3.9906880855560303, "mode": "train", "epoch": 28}
{"loss": 0.5481914281845093, "acc": 0.7339844107627869, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.4924105703830719, "acc": 0.7675501108169556, "length": 3.990511655807495, "mode": "train", "epoch": 29}
{"loss": 0.5443603992462158, "acc": 0.7414062023162842, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.48728469014167786, "acc": 0.7724844217300415, "length": 3.992002010345459, "mode": "train", "epoch": 30}
{"loss": 0.5354938507080078, "acc": 0.7498046159744263, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.48153892159461975, "acc": 0.7766353487968445, "length": 3.992002248764038, "mode": "train", "epoch": 31}
{"loss": 0.5274470448493958, "acc": 0.7554687857627869, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.4734641909599304, "acc": 0.781663715839386, "length": 3.9925873279571533, "mode": "train", "epoch": 32}
{"loss": 0.5431326627731323, "acc": 0.7464843392372131, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.46485981345176697, "acc": 0.7853114008903503, "length": 3.993705987930298, "mode": "train", "epoch": 33}
{"loss": 0.5182387828826904, "acc": 0.7623047232627869, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.46234044432640076, "acc": 0.7888314127922058, "length": 3.994391679763794, "mode": "train", "epoch": 34}
{"loss": 0.5270969867706299, "acc": 0.7537108063697815, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.45617976784706116, "acc": 0.7941255569458008, "length": 3.9929215908050537, "mode": "train", "epoch": 35}
{"loss": 0.5065291523933411, "acc": 0.7755860090255737, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.44795840978622437, "acc": 0.7966666221618652, "length": 3.9937164783477783, "mode": "train", "epoch": 36}
{"loss": 0.515466570854187, "acc": 0.7677735090255737, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.44412994384765625, "acc": 0.8012982606887817, "length": 3.9950764179229736, "mode": "train", "epoch": 37}
{"loss": 0.5094435811042786, "acc": 0.7730469107627869, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.4392608106136322, "acc": 0.8023365139961243, "length": 3.9937870502471924, "mode": "train", "epoch": 38}
{"loss": 0.5029140710830688, "acc": 0.7726562023162842, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.4362604320049286, "acc": 0.8049876093864441, "length": 3.9950339794158936, "mode": "train", "epoch": 39}
{"loss": 0.5056918859481812, "acc": 0.7685547471046448, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.4307849407196045, "acc": 0.8067901134490967, "length": 3.9950716495513916, "mode": "train", "epoch": 40}
{"loss": 0.5040633678436279, "acc": 0.7705078721046448, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.42520901560783386, "acc": 0.8132330775260925, "length": 3.996107578277588, "mode": "train", "epoch": 41}
{"loss": 0.4940495491027832, "acc": 0.7742187976837158, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.42427513003349304, "acc": 0.8147583603858948, "length": 3.995819330215454, "mode": "train", "epoch": 42}
{"loss": 0.49291062355041504, "acc": 0.7738280892372131, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.421440064907074, "acc": 0.8123003840446472, "length": 3.9958133697509766, "mode": "train", "epoch": 43}
{"loss": 0.4981585741043091, "acc": 0.7757812142372131, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.4186713993549347, "acc": 0.8144248723983765, "length": 3.997073173522949, "mode": "train", "epoch": 44}
{"loss": 0.4920429289340973, "acc": 0.7845702767372131, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.414288729429245, "acc": 0.8175743222236633, "length": 3.9964940547943115, "mode": "train", "epoch": 45}
{"loss": 0.49396848678588867, "acc": 0.7767577767372131, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.4099748134613037, "acc": 0.818174421787262, "length": 3.9972732067108154, "mode": "train", "epoch": 46}
{"loss": 0.4998076558113098, "acc": 0.7662109732627869, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.4098985493183136, "acc": 0.8195109963417053, "length": 3.9970691204071045, "mode": "train", "epoch": 47}
{"loss": 0.4866563081741333, "acc": 0.7761719226837158, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.40615659952163696, "acc": 0.8212248682975769, "length": 3.9980437755584717, "mode": "train", "epoch": 48}
{"loss": 0.49433302879333496, "acc": 0.7832030653953552, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.40503406524658203, "acc": 0.8206960558891296, "length": 3.9979474544525146, "mode": "train", "epoch": 49}
{"loss": 0.49856626987457275, "acc": 0.7710937857627869, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.40465158224105835, "acc": 0.8250225186347961, "length": 3.9979214668273926, "mode": "train", "epoch": 50}
{"loss": 0.4911404848098755, "acc": 0.7748046517372131, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.3951862156391144, "acc": 0.8263818621635437, "length": 3.99859881401062, "mode": "train", "epoch": 51}
{"loss": 0.4897269606590271, "acc": 0.7830076813697815, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.399649441242218, "acc": 0.824640154838562, "length": 3.9968926906585693, "mode": "train", "epoch": 52}
{"loss": 0.4873967468738556, "acc": 0.7824218273162842, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.39682868123054504, "acc": 0.8250358700752258, "length": 3.9986977577209473, "mode": "train", "epoch": 53}
{"loss": 0.48619574308395386, "acc": 0.7820312976837158, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.3939720690250397, "acc": 0.8270745277404785, "length": 3.9983479976654053, "mode": "train", "epoch": 54}
{"loss": 0.48980116844177246, "acc": 0.7742186188697815, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.39028143882751465, "acc": 0.8265566229820251, "length": 3.998375654220581, "mode": "train", "epoch": 55}
{"loss": 0.4849994480609894, "acc": 0.7871093153953552, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.38941439986228943, "acc": 0.8270249366760254, "length": 3.998546600341797, "mode": "train", "epoch": 56}
{"loss": 0.4875507652759552, "acc": 0.7798828482627869, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.38662484288215637, "acc": 0.8295907974243164, "length": 3.99816632270813, "mode": "train", "epoch": 57}
{"loss": 0.4902728796005249, "acc": 0.7810546159744263, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.38625437021255493, "acc": 0.8337458968162537, "length": 3.9982447624206543, "mode": "train", "epoch": 58}
{"loss": 0.47909387946128845, "acc": 0.7888670563697815, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.38421592116355896, "acc": 0.8298950791358948, "length": 3.998781204223633, "mode": "train", "epoch": 59}
{"loss": 0.4867950677871704, "acc": 0.7828125357627869, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.38269686698913574, "acc": 0.8308257460594177, "length": 3.9986414909362793, "mode": "train", "epoch": 60}
{"loss": 0.4779241383075714, "acc": 0.7871093153953552, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.752666711807251
